concepts/n-gram.md

```diff
@@ -1,7 +1,19 @@
 ---
 grand_parent: Resources
 parent: Concepts
-layout: coming_soon
 title: N-gram
-description:
+description: A short sequence of word types
 ---
+
+An **n-gram** is a short sequence of word types.
+`n` is the number of words in the sequence, for example, a 2-gram has two word types.
+N-grams are commonly used in [language models](/concepts/language-model), for example in a [n-gram maximum likelihood estimate](/concepts/language-model#n-gram-maximum-likelihood-estimate).
+N-grams are also used in language identification and translation models for [statistical machine translation](/approaches/statistical-machine-translation.md).
+
```

Comment:
We have an open PR on `Metrics`, do you think it would be interesting to hyperlink this article to `n-gram-based metrics`?

Reply:
Added